### PR TITLE
Update h5vcc_metrics.idl interface

### PR DESCRIPTION
**Pull request by joeltine**
These are changes based on conversation in
http://go/cobalt-telemetry-impl.

b/280094891

Change-Id: Iec2b4b89c931a4602311f881ad678e4dfb0d4951

### CODE REVIEW

**Comment by joeltine** [comment]
Friendly ping on a review plz

File: cobalt/browser/metrics/cobalt_metrics_service_client_test.cc [hunk 1]
```diff
@@ -105,6 +105,15 @@ TEST_F(CobaltMetricsServiceClientTest, TestStubbedSystemFields) {
...
 }
 
+TEST_F(CobaltMetricsServiceClientTest, UploadIntervalCanBeOverriden) {
+  ASSERT_EQ(client_->GetStandardUploadInterval().InSeconds(), 300);
+  client_->SetUploadInterval(42);
+  ASSERT_EQ(client_->GetStandardUploadInterval().InSeconds(), 42);
+  client_->SetUploadInterval(UINT32_MAX);
+  // UINT32_MAX is the sentinel value to revert back to the default of 5 min.
+  ASSERT_EQ(client_->GetStandardUploadInterval().InSeconds(), 300);
+}
```
Comment (sideb0ard) [inline cobalt/browser/metrics/cobalt_metrics_service_client_test.cc:115]: how about a test to set upload interval to zero? The IDL says it should be ignored.

**Review by sideb0ard — APPROVED** [review]
yeah, looks straight forwards and aligned to discussion in impl doc. Left a suggestion for additional test if possible but lgtm.

File: cobalt/browser/metrics/cobalt_metrics_service_client_test.cc (hunk 1 above)
Comment (joeltine) [reply cobalt/browser/metrics/cobalt_metrics_service_client_test.cc:115]: Great call. Added.

**Pull request merged**